docs/tutorials/using_apis/connectivity/light-control.md

```diff
@@ -63,95 +99,69 @@ An example of the `mbed_app.json` file:
+To run a 6LoWPAN-ND network, you need a working RF driver for Nanostack. This example uses the Atmel AT86RF233 by default. Place the shield on top of your board.
 
-To change the RF driver, modify the `mbed_app.json` file by setting preferred RF driver `provide_default` value to true, For example, to use MCR20a RF driver: 
+To change the RF driver, set the preferred RF driver `provide_default` value to `true` in `mbed_app.json`. For example, to use the MCR20a RF driver:
 
 ```json
 "atmel-rf.provide-default": false,
 "mcr20a.provide-default": true
 ```
 
-## Compile the application
+## Compiling and flashing the application
 
-```
-mbed compile -m K64F -t GCC_ARM
-```
-
-A binary is generated in the end of the build process.
-
-## Connect the RF shield to the board
+1. To compile the application:
 
-This example uses the Atmel AT86RF233, which you can [purchase](https://firefly-iot.com/product/firefly-arduino-shield-2-4ghz/). Place the shield on top of your board, and power it on.
+    ```
+    mbed compile -m K64F -t GCC_ARM
+    ```
 
-## Program the target
-
-Drag and drop the binary to the target to program the application.
+1. A binary is generated at the end of the build process.
+1. Connect the board to your computer, and power on the board.
+1. Drag and drop the binary to the board (shown as storage device on the computer) to program the application.
 
 ## Update the firmware of the border router
+<!--do I have to do this? do I have to do this at this point in the process, or can I put it as one of the requirements that should have been listed earlier in the doc?-->
 
-This example supports the following border router:
-
-[Nanostack-border-router](https://github.com/ARMmbed/nanostack-border-router).
+This example supports the [Nanostack-border-router](https://github.com/ARMmbed/nanostack-border-router).
 
-The border router supports static and dynamic backhaul configuration. The static configuration is good for testing, but the dynamic one works if your network infrastructure is supplying an IPv6 address. Make sure that you use the appropiate mode.
+The border router supports static and dynamic backhaul configuration. The static configuration is good for testing, but the dynamic one works <!--this phrasing is misleading, I think. why are testing and dynamic allocation contradictory?--> if your network infrastructure supplies an IPv6 address. Make sure that you use the appropriate mode.<!--is this what I'm supposedly updating?-->
 
-Remember to connect the Ethernet cable between the border router and your router. Then power on the board.
+Remember to connect the Ethernet cable between the border router and your router<!--router and router? router and board?-->. Then power on the board.<!--it was already powered on to flash it. did I disconnect it from the computer at some point? did I power it off?--><!--is this really part of "update your router"? won't it be equally valid if my border router was already correct?-->
```

Comment:
@SeppoTakalo @jarlamsa Can you answer this?

Reply:
Probably the emphasis was to remind the user to have the Ethernet cable between the board and router. "Then power cycle your board" might be better after that.